### PR TITLE
95548_create_category

### PR DESCRIPTION
**Pull request by nvductk**
Create category nested set model
Rubocop results:
40 files inspected, no offenses detected

### CODE REVIEW

File: app/views/admin/categories/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <a class="waves-effect waves-light btn" href="#modal1"><%= t".add" %></a>
+  <%= form_tag admin_categories_path , method: :post,
+    id: "add-category-form" do %>
+  <div id="modal1" class="modal">
```
Comment (Kominam) [inline app/views/admin/categories/index.html.erb]: indent a oi

File: app/views/admin/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          class: "form-control" %>
+     </div>
+    </div>
+    <div class="modal-footer">
+      <button class="modal-action modal-close waves-effect waves-green btn-flat" data-dismiss="modal">
+      <%= t ".cancel" %>
+       <button class="modal-action modal-close waves-effect waves-green btn-flat">
+       <%= submit_tag t(".add")%>
+    </div>
+  </div>
```
Comment (Kominam) [inline app/views/admin/categories/index.html.erb]: đây nữa

File: app/views/admin/categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <a class="waves-effect waves-light btn" href="#modal1"><%= t".add" %></a>
+  <%= form_tag admin_categories_path , method: :post,
+    id: "add-category-form" do %>
```
Comment (Kominam) [inline app/views/admin/categories/index.html.erb]: indent block code trong form a ơi

**Comment by nvductk** [comment]
Create category nested set model
Rubocop results:
40 files inspected, no offenses detected

**Pull request merged**